File: AFMobile/AFAndroid/app/src/main/java/cz/cvut/fel/matyapav/afandroid/ValidationRule.java
package cz.cvut.fel.matyapav.afandroid;

/**
 * Created by Marcelka on 25.12.2015.
 */
public class ValidationRule {

    private String validationType;
    private String value;

    public ValidationRule() {
    }

    public String getValidationType() {
        return validationType;
    }

    public void setValidationType(String validationType) {
        this.validationType = validationType;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }

    @Override
    public String toString() {
        return "ValidationRule{" +
                "validationType='" + validationType + '\'' +
                ", value='" + value + '\'' +
                '}';
    }
}

File: AFMobile/AFWindowsPhone/AFWindowsPhone/components/parts/ValidationRule.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AFWindowsPhone.builders.components.parts
{
    class ValidationRule
    {
        private String validationType;
        private String value;

        public ValidationRule()
        {
        }

        public String getValidationType()
        {
            return this.validationType;
        }

        public void setValidationType(String validationType)
        {
            this.validationType = validationType;
        }

        public String getValue()
        {
            return this.value;
        }

        public void setValue(String value)
        {
            this.value = value;
        }
    }
}
